### ISSUE TITLE
[REQUEST] nd2_to_tiff -- write log

### ISSUE THREAD

**Issue by elgw** [ggirelli/radiantkit#51]
It would be great if the log messages (like below) also could be written to a file in the newly created folder.

```
06/05/2020 09:58:46 [P25133:nd2_to_tiff:run] INFO: Enforcing a deltaZ of 0.300 um.
06/05/2020 09:58:46 [P25133:conversion:log_details] INFO: Found 19 field(s) of view, with 7 channel(s).
06/05/2020 09:58:46 [P25133:conversion:log_details] INFO: Channels: ['ir800', 'a700', 'a488', 'cy5', 'tmr', 'a594', 'dapi'].
06/05/2020 09:58:46 [P25133:conversion:log_details] INFO: XYZ size: 1024 x 1024 x 81
06/05/2020 09:58:47 [P25133:conversion:log_details] INFO: Delta Z value(s): {0.2, 0.225, 0.175}
06/05/2020 09:58:47 [P25133:nd2_to_tiff:run] INFO: Output directory: 'iAM337_20190830_001'
```

**Comment by ggirelli** [comment]
This will be fixed in #100

**Comment by ggirelli** [comment]
A log is now automatically generated in the output directory by using `rich`.
